daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Engine.scala

```diff
@@ -123,6 +127,7 @@ final class Engine {
     * create or an exercise choice.
     */
   def reinterpret(
+      seed: Option[crypto.Hash],
```

Comment:
as above: more descriptive name and comments.

Reply:
add description in the comment above